worlds/clique/__init__.py

```diff
@@ -27,71 +24,70 @@ class CliqueWebWorld(WebWorld):
 
 
 class CliqueWorld(World):
-    """The greatest game ever designed. Full of exciting gameplay!"""
+    """The greatest game of all time."""
 
     game = "Clique"
-    data_version = 2
+    data_version = 3
     web = CliqueWebWorld()
     option_definitions = clique_options
-
-    # Yes, I'm like 12 for this.
-    location_name_to_id = {
-        "The Big Red Button":   69696969,
-        "The Item on the Desk": 69696968,
-    }
-
-    item_name_to_id = {
-        "Feeling of Satisfaction": 69696969,
-        "Button Activation":       69696968,
-    }
+    location_name_to_id = location_table
+    item_name_to_id = item_table
 
     def create_item(self, name: str) -> CliqueItem:
-        return CliqueItem(name, ItemClassification.progression, self.item_name_to_id[name], self.player)
+        return CliqueItem(name, item_data_table[name].type, item_data_table[name].code, self.player)
```

Comment:
shouldn't this raise if someone plandoes "Button Activation" with "hard_mode" disabled?
```suggestion
        return CliqueItem(name, item_data_table[name].type, item_data_table[name].code, self.player)
```

Reply:
Are you asking if it should raise or it does if that happens? If someone plandos Button Activation it would just create a useless item.

Worst case, it will cause a generation failure (or ignore for `force: false` and `force: null` if it's not possible like in a single player game.

Reply:
that was a play on the attribute name `can_create`. if `can_create` returns false, it should crash, no?

Reply:
Nope, `can_create` is only referenced in `create_items` for normal items.

Reply:
i know. i would expect it to throw an exception or divide by 0 when being called from plando

Reply:
I guess I disagree.